files/k8s-resource/calico-policy-only.yaml

```diff
@@ -320,6 +321,7 @@ spec:
   template:
     metadata:
       labels:
+        app.kubernetes.io/name: calico-node
```

Comment:
This is required so that we don't have separate waits with different labels depending on whether it's calico policy-only or full.